src/core/core.datasetController.js

```diff
@@ -1158,11 +1169,10 @@ DatasetController.prototype._datasetElementOptions = [
 ];
 
 /**
- * Data element option keys to be resolved in _resolveDataElementOptions.
+ * Data element option keys to be resolved in resolveDataElementOptions.
  * A derived controller may override this to resolve controller-specific options.
  * The keys defined here are for backward compatibility for legend styles.
  * @type {string[]|object}
- * @private
  */
 DatasetController.prototype._dataElementOptions = [
```

Comment:
We should make this `@protected` and remove the underscore? (same with `_datasetElementOptions`)

Reply:
ts does not like them protected (not in class). Could probably do that in #7125 
Removed the underscore though.